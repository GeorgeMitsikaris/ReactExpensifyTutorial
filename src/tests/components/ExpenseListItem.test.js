import React from 'react';
import {shallow} from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import { expenseOne } from '../fixtures/expenses';

test('should render ExpenseListItem' , () => {
     const wrapper = shallow(<ExpenseListItem {...expenseOne}/>)
     expect(wrapper).toMatchSnapshot();
})