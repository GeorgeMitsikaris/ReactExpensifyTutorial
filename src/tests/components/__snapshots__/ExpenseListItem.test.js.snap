// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      description for rent
    </h3>
  </Link>
  <p>
    0.10
     Euros -
    January 1st, 1970
  </p>
</div>
`;
